kiln/devnets/goerlishadow-4.vars

```diff
@@ -0,0 +1,21 @@
+DEVNET_NAME=goerlishadow-4
+
+GETH_IMAGE=parithoshj/geth:master
+NETHERMIND_IMAGE=nethermindeth/nethermind:kiln_0.12
+ETHEREUMJS_IMAGE=g11tech/ethereumjs:kiln
+LODESTAR_IMAGE=chainsafe/lodestar:next
+CONFIG_GIT_DIR=goerli-shadow-fork-4
```

Comment:
Hopefully this DIR is matching with the corresponding dir in https://github.com/eth-clients/merge-testnets which I guess its matching :+1: